scripts/enter.sh

```diff
@@ -7,6 +7,11 @@ CACHE_DIR="${CACHE_DIR:-$HOME/hassos-cache}"
 ARGS="$*"
 COMMAND="${ARGS:-bash}"
 
+if [ "$BUILDER_UID" -eq "0" ] || [ "$BUILDER_GID" == "0" ]; then
+  echo "ERROR: Please use a non-root with sudo rights for running this script."
```

Comment:
Reads a bit hard, maybe:

```suggestion
  echo "ERROR: Please run this script as a regular (non-root) user with sudo privileges."
```